src/popover/constants.js

```diff
@@ -42,7 +42,7 @@ export const POPOVER_MARGIN = 8;
 
 export const ARROW_SIZE = 6;
 
-export const ANIMATE_OUT_TIME = 500;
```

Comment:
Should it be overwritten just in the select's popover usage case instead of turning it off for the popover globally? This would affect the tooltip also.

Reply:
We could. Personally I am not a big fan of fade-in/fade-out animations since it often makes application looking sluggish but up to you. Also, should it be configurable then?

Reply:
I think making it configurable makes perfect sense - should we add this to the theme?

Reply:
I added it as new Popover prop for now. I think the theme makes sense once we would like to use the same value across multiple components.